domain-management/src/main/java/org/jboss/as/domain/management/parsing/ManagementXml.java

```diff
@@ -47,7 +47,11 @@ public static ManagementXml newInstance(Namespace namespace, ManagementXmlDelega
             case 3:
                 return new ManagementXml_Legacy(namespace, delegate);
             default:
-                return new ManagementXml_4(namespace, delegate);
+                if(namespace.getMinorVersion() < 2) {
+                    return new ManagementXml_4(namespace, delegate);
+                } else {
+                    return new ManagementXml_4_2(namespace, delegate);
```

Comment:
As I said in the main comments this should all be handled in ManagementXml_4 - a minor version doesn't need a complete clone.